Laba1-0/src/main/kotlin/laba5/1.kt

```diff
@@ -9,8 +9,6 @@ package laba5
 val numbers = listOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
 
 fun main() {
-    var n = numbers.filter { it % 2 == 0 }
-    println(n)
-    n = numbers.filter { it % 2 == 0 }.map { it * 2 }
+    val n = numbers.filter { it % 2 == 0 }.filter { it % 2 == 0 }.map { it * 2 }
```

Comment:
в котлине принято такие цепочки писать на новой строке
```suggestion
    val n = numbers
    .filter { it % 2 == 0 }
    .filter { it % 2 == 0 }
    .map { it * 2 }
```

ну ток форматирование поправить

Reply:
не оч понятно зачем два раза фильтровать % 2 :D